README.md

```diff
@@ -8,3 +8,4 @@ Hello my name is John and I'm wondering how much content I need to write for it
  ok well that's what I had to say. Cool to see the numbers only change based off of times I press "enter".
 print = "that".
 type = blah
```

Comment:
why was type set to blah here? I don't think we previously defined that list.

Reply:
ohhhhh, I set it to blah because I was treating it like _____ when it was actually _____. Good callout, John. Thanks.